### PR TITLE
Feature/add polygon config

### PR DESCRIPTION
**Pull request by denciu**
- added Polygon basic config
- added/updated Polygon related tokens
- updated Sushiswap config to support Polygon
- added Quickswap to UniswapV2RoutablePlatforms
- added DFYN to UniswapV2RoutablePlatforms ( by [jorgedclopes](https://github.com/jorgedclopes) )

### CODE REVIEW

**Review by adamazad — APPROVED** [review]
Nice! I verified all the addresses. Looks good to go!

File: src/entities/trades/routable-platform/uniswap-v2-routable-platform.ts [hunk 1]
```diff
@@ -97,6 +101,14 @@ export class UniswapV2RoutablePlatform extends RoutablePlatform {
     initCodeHash: '0x4955fd9146732ca7a64d43c7a8d65fe6db1acca27e9c5b3bee7c3abe5849f441',
     defaultSwapFee: _30
   })
+  public static readonly QUICKSWAP = new UniswapV2RoutablePlatform({
+    chainIds: [ChainId.POLYGON],
+    name: 'Quickswap',
+    factoryAddress: {[ChainId.POLYGON]: QUICKSWAP_FACTORY_ADDRESS},
+    routerAddress: {[ChainId.POLYGON]: QUICKSWAP_ROUTER_ADDRESS},
+    initCodeHash: '0x96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f',
```
[the remark this answers is not in the document]
Comment (denciu) [reply src/entities/trades/routable-platform/uniswap-v2-routable-platform.ts:109]: https://github.com/QuickSwap/QuickSwap-sdk/blob/master/src/constants.ts

File: src/constants.ts [hunk 2]
```diff
@@ -47,31 +48,35 @@ export const FACTORY_ADDRESS: { [chainId in ChainId]: string } = {
   [ChainId.RINKEBY]: coreRinkeby.factory,
   [ChainId.ARBITRUM_ONE]: coreArbitrumOne.factory,
   [ChainId.ARBITRUM_RINKEBY]: coreArbitrumRinkebyTestnet.factory,
-  [ChainId.XDAI]: coreXDai.factory
+  [ChainId.XDAI]: coreXDai.factory,
+  [ChainId.POLYGON]: '0x0000000000000000000000000000000000001234'
```
[the remark this answers is not in the document]
Comment (denciu) [reply src/constants.ts:52]: I think I copied it from `SWPR_CONVERTER_ADDRESS.[ChainId.Mainnet]`, I can change it to `ZERO_ADDRESS`

[the remark this answers is not in the document]
Comment (denciu) [reply src/constants.ts]: Done

**Review by luzzif — APPROVED** [review]

**Review by adamazad — APPROVED** [review]

File: src/constants.ts [hunk 3]
```diff
@@ -81,6 +86,7 @@ export const SWPR_CONVERTER_ADDRESS: { [chainId in ChainId]: string } = {
   [ChainId.ARBITRUM_RINKEBY]: '0x0000000000000000000000000000000000001234',
   [ChainId.ARBITRUM_ONE]: '0x2b058af96175A847Bf3E5457B3A702F807daDdFd',
   [ChainId.XDAI]: '0x0000000000000000000000000000000000001234',
+  [ChainId.POLYGON]: ZERO_ADDRESS
```
[the remark this answers is not in the document]
Comment (denciu) [reply src/constants.ts]: Done

**Review by adamazad — APPROVED** [review]
Thank you!

**Pull request merged**